Alameen688/mydiary-spa
# #162304717 User should be able to see a landing page

#### What does this PR do?
Implements landing page
#### Description of Task to be completed?
-  Both authorized and unauthorized users should be able to view the landing page
-  The landing page should contain a link to log in and register page
-  Add files that mock css imports and file imports for tests.
#### How should this be manually tested?
- Run `npm run dev`
- Go to `localhost:3000`
- You should see a landing page that looks like the screenshot provided below
#### Any background context you want to provide?

#### What are the relevant pivotal tracker stories?
[#162304717](https://www.pivotaltracker.com/story/show/{#162304717})

#### Screenshots (if appropriate)
![screen shot 2018-11-29 at 12 59 37 pm 2](https://user-images.githubusercontent.com/9039613/49220515-b630c300-f3d6-11e8-90ef-0033aa47714c.png)

#### Questions:
